source/features/linkify-commit-sha.js

```diff
@@ -0,0 +1,13 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import {wrap} from '../libs/utils';
+
+export default function () {
+	let commitUrl = location.pathname.replace(/\/$/, '');
+	commitUrl = commitUrl.replace(/\/pull\/\d+\/commits/, '/commit');
+
+	const el = select('.sha-block:not(.patch-diff-links) .sha');
```

Comment:
This can probably be just `.sha.user-select-contain`

Reply:
Addressed in f617cdd.